changelog.txt

```diff
@@ -21,6 +21,7 @@
 * Update - Bump minimum supported version of WooCommerce from 5.4 to 5.5.
 * Update - Bump minimum required version of WooCommerce from 4.0 to 4.4.
 * Fix - Add credit card on My Account using other payment gateways does not show "Your card number is incomplete" error
+* Fix - Appearance of upload file buttons inside challenge dispute page
```

Comment:
I think this isn't required as there should be a process for consolidating the changelog before release, and I'm not sure how people are going about this now in this repository, but the majority of PRs I review often add the same changelog entry to readme.txt as well, so that might be a good idea.

Also, most changelog entries have a `.` at the end, so it's nice to keep it consistent.

The above is slightly pedantic, so feel free to ignore.